packages/at_client_mobile/lib/src/enrollment/at_enrollment_service_impl.dart

```diff
@@ -287,8 +340,16 @@ class AtEnrollmentServiceImpl implements AtEnrollmentService {
     return atEncryptionResult.result;
   }
 
-  @override
-  Future<AtEnrollmentResponse> manageEnrollmentApproval(
+  Future<AtEnrollmentResponse> approve(
+      AtEnrollmentRequest atEnrollmentRequest) {
+    return _manageEnrollmentApproval(atEnrollmentRequest);
+  }
+
+  Future<AtEnrollmentResponse> deny(AtEnrollmentRequest atEnrollmentRequest) {
+    return _manageEnrollmentApproval(atEnrollmentRequest);
+  }
+
```

Comment:
> Since, at_auth package is used internally, can we do a breaking change to rename the methods `manageEnrollmentApproval` to approve and deny (similar to what we did in at_client_mobile)?

Don't need a breaking change; deprecate the existing method and add `approve` and `deny` methods